server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -1194,7 +1194,8 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
             timestamp: message.timestamp,
             traces: message.operation.traces,
             type: message.operation.type,
-        };
+            compression: (message.operation as any).compression,
```

Comment:
Please add "// back-compat" comment here. It's not the best (i.e. it's less about compat), but we do scan periodically repo for these comments and remove code that is no longer needed (including unneeded casts). Or open an issue to track removal of it.